### DISCUSSION TITLE
英特爾新FakeCatcher偵測技術，能以96%準確度即時辨識Deepfake影片

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#885] [General]
FakeCatcher是用來辨識影片中的人像是否為AI合成。其利用判斷影片人像的血流，來證明影片非合成。因為人類的心臟跳動泵血作用發生時，靜脈的顏色會發生改變，FakeCatcher收集臉部的血流訊號，並將這些訊號轉換成時空圖（Spatiotemporal Map），用深度學習演算法判斷影片真偽。我覺得可能人類需要在平面圖領域也有這樣的技術，去規範AI在繪圖領域的崛起。
https://www.ithome.com.tw/news/154292

**Comment by StevenWanglolz** [comment]
真的很希望這項技術能持續精進，以免有更多人受害!
我最近看了一部有關Deepfake的Ted Talk，裡面提到了Deepfake對於個體和社會整體的威脅性，還有該怎麼避免此類事情發生，有中文字幕不用擔心~

影片連結: https://www.ted.com/talks/danielle_citron_how_deepfakes_undermine_truth_and_threaten_democracy

**Comment by Chiway1229** [comment]
讓我想到之前一個名為小玉的實況主就爆出使用deepfake來獲利的醜聞
希望台灣能盡快推進相關法令

**Comment by ching020721** [comment]
希望這類型的技術能發展起來，避免先前的犯罪事件再次重演

**Comment by Kotori0501** [comment]
這項技術真的好強，僅僅一張圖片就可以判斷臉上的血流，感覺有點酷。
希望能持續精進這項技術
這樣就不會有更多人受害了

**Comment by HuangHsinTzu** [comment]
太棒了! 好多人(尤其是藝人)深受Deepfake其害
這項技術可以一定程度的控制Deepfake影片的氾濫
但感覺還是不能太依賴這項技術
因為不能排除非法人士有一天研發出一種技術可以騙過這個辨識系統的可能性

**Comment by CPCpyt** [comment]
感覺這個技術可以幫助一些受Deepfake所害的名人討回公道，
並且在法庭上能提供有力的證據。
